serverless.yml

```diff
@@ -57,6 +57,13 @@ functions:
           path: /movies
           method: get
           cors: true
+  movieById:
+    handler: src/handlers/movies/movieById.handler
+    events:
+      - http:
+          path: movie/{id}
```

Comment:
I prefer to keep the plural `/movies/{id}` here.

AFAIK, there's no real RESTful rule about this naming convention, I just think it makes more sense because the path  you're creating means it's a resource of the `movies` collection which was previously set as plural from the beginning.

I just noticed I created the `genre/movies` path (first singular) - I will change that to keep the pattern.